ecosystem/sep-0031.md

```diff
@@ -68,6 +68,30 @@ All endpoints respond with content type:
 
 - `application/json`
 
+## Asset Conversions and Liquidity Strategies
+
+This protocol can be used if the Sending and Receiving Client wish to send and receive the same asset, but this is typically not the case. It is more common for the Sending Client to provide the Sending Anchor with a particular asset (the source asset) and have the Receiving Client receive a different asset (the destination asset).
+
+In this case, the source asset must convert into the receiving asset at some point prior to delivering funds to the Receiving Client. This conversion must be done by either the Sending Anchor, the Stellar Network, or the Receiving Anchor. The subsections below outline the stratgies used for each method.
+
+### Sending Anchor Asset Conversion
+
+The Sending Anchor can collect the source asset from the Sending Client and send the destination asset to the Receiving Anchor. 
+
+This approach requires the Sending Anchor to provide an exchange rate, or quote, to the Sending Client. It also requies the Sending Anchor to hold a balance of the destination asset on Stellar. The Sending Anchor would use Payment operations to send the quoted amount of the destination asset to the Receiving Anchor.
+
+### Stellar Network Asset Conversion
+
+Instead of holding a balance of the destination asset on Stellar, the Sending Anchor can instead us Path Payments to convert the source asset to the destination asset on the Stellar Network. 
+
+This approach requires a sufficiently liquid market to exist between the source and destination assets on the Stellar Decentralized Exchange (SDEX). Note that the exchange rate between the two assets is dynamic due to market fluctuations on the SDEX.
+
+### Receiving Anchor Asset Conversion
+
+Finally, the Receiving Anchor can receive payments of the source asset on Stellar and deliver payments of the destination asset to the Receiving Client. 
+
+This requires the Receiving Anchor to implement the [SEP-38 Anchor RFQ API](sep-0038.md), which enables the Receiving Anchor to provide quotes to the Sending Anchor. These quotes can be [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) or [firm](https://www.investopedia.com/terms/f/firmquote.asp). If a firm quote is used, the Stellar transaction _must_ be submitted to the Stellar network prior to the quote's expiration. More specifically, the `created_at` timestamp returned in the [Horizon GET /transactions/:id](https://developers.stellar.org/api/resources/transactions/) request must be earlier than the quote's expiration.
+
```

Comment:
This is really great. We should have @phil-stellar and @lisaclaire828 take a look as well. I would consider the following renaming of strategies to reflect *when* the asset conversion happens: 

`Sending Anchor Asset Conversion` -> `Pre-payment asset conversion`

`Stellar Network Asset Conversion`-> `Real time asset conversion` 

`Receiving Anchor Asset Conversion` -> `Post-payment asset conversion`

Reply:
I think those names somewhat obfuscate who is responsible for asset conversion, and if I read them without context of what's in each section I don't really know what they mean.

Reply:
I agree with Leigh, I think the _who_ is more important than the _when_